docs/source/NEXUS.rst

```diff
@@ -33,10 +33,9 @@ Important information about NEXUS code:
 Tags
 ------------
 A list of the different NEXUS `tags <https://github.com/next-exp/nexus/wiki/Tags>`_ can be found on the Wiki.
-Starting from nexus tag v7_00_00, the following versions are needed:
+Starting from nexus tag v7_00_00, Geant4 version >= 11.0 is needed:
```

Comment:
The colon at the end doesn't make sense now. And I think the two sentences can now be in the same paragraph.